kubetest2-gke/deployer/build/gke_make.go

```diff
@@ -112,6 +116,27 @@ func (gmb *GKEMake) Stage(version string) error {
 	if err := gmb.runWithActions(os.Stdout, os.Stderr, []gkeBuildAction{stage}, args...); err != nil {
 		return err
 	}
+
+	if gmb.UpdateLatest {
+		r := regexp.MustCompile(gkeMinorVersionRegex)
+		m := r.FindStringSubmatch(version)
+		var fName string
+		if len(m) < 2 {
+			klog.Warningf("can't find the minor version of %s, defaulting to latest.txt", version)
+			fName = "latest.txt"
+		} else {
+			minor := m[1]
+			fName = fmt.Sprintf("latest-%s.txt", minor)
+		}
+		pushCmd := fmt.Sprintf("gsutil cp - %s/%s", gmb.StageLocation, fName)
+		cmd := exec.Command(pushCmd)
```

Comment:
won't this put everything as the command without any arguments?

you might want exec.RawCommand()

Reply:
Done